ckanext/xloader/plugin.py

```diff
@@ -208,3 +229,50 @@ def get_helpers(self):
             xloader_helpers.xloader_status_description,
             'is_xloader_format': xloader_helpers.is_xloader_format,
         }
+
+
+def _should_remove_unsupported_resource_from_datastore(res_dict_or_obj):
+    if not toolkit.asbool(toolkit.config.get('ckanext.xloader.limit_datastore_tables', False)):
```

Comment:
A better config option name, remember to document it.

```suggestion
    if not toolkit.asbool(toolkit.config.get('ckanext.xloader.clean_datastore_tables', False)):
```